python/lib/scheduler/dmod/scheduler/job/job.py

```diff
@@ -1,6 +1,6 @@
 from abc import ABC, abstractmethod
```

Comment:
Version bump scheduler?

Reply:
Yeah, will change.  See above for monitor package.